egov-indexer/citizen-feedback-indexer.yml

```
@@ -49,7 +49,7 @@ ServiceMaps:
            - inJsonPath: $.additionalDetails.localityCode
              outJsonPath: localityCode
            externalUriMapping:
-           - path: http://egov-location.pucar-prod:8080/egov-location/location/v11/boundarys/_search
+           - path: http://egov-location.egov:8080/egov-location/location/v11/boundarys/_search
```

yamllint (1.35.1)

[error] wrong indentation: expected 13 but found 11

(indentation)